src/context/fetch.ts

```diff
@@ -26,6 +28,23 @@ const createFetchRequestParameters = (input: MockedRequest): RequestInit => {
     return requestParameters
   }
 
+  // When the user made their original request, the browser added a boundary e.g.
+  // "content-type": multipart/form-data; boundary=----WebKitFormBoundarygyGKnRF8C9LT0BhB"
+  // If we now reuse this string as a user-provided header, the form-data will break and no payload will be send
+  // https://community.cloudflare.com/t/cannot-seem-to-send-multipart-form-data/163491/2
+  if (
+    input.headers.get('content-type')?.includes('multipart/form-data') &&
+    typeof body === 'object'
+  ) {
+    input.headers.delete('content-type')
```

Comment:
Is it a problem that I am mutating the input?

It might be better to make a copy of the headers and mutate the copy:

```ts
const requestParameters: RequestInit = {
    ...input,
    headers: new Headers(input.headers),
    body: undefined,
  }
```

Would I use `Headers` or `HeadersPolyfill` for that?